README.md

```diff
@@ -322,3 +322,44 @@ You can decode custom types the same way, as long as the type also conforms to
 `JSONDecodable`.
 
 For more examples on how to use Argo, please check out the tests.
+
+Contributing
+------------
+
+See the [CONTRIBUTING] document.
+Thank you, [contributors]!
+
+[CONTRIBUTING]: CONTRIBUTING.md
+[contributors]: https://github.com/thoughtbot/Argo/graphs/contributors
+
+Need Help?
+----------
+
+We offer 1-on-1 coaching. We can help you with functional programming in Swift,
+get started writing integration tests, and convert from Objective C to Swift.
```

Comment:
Should this be unit tests instead of integration tests? My understanding is that the xcoders team is lukewarm on current iOS integration testing solutions.